linters/renovate/plugin.yaml

```diff
@@ -17,8 +10,7 @@ lint:
   definitions:
     - name: renovate
       files: [renovate-config]
-      runtime: node
-      package: renovate
+      tools: [renovate]
```

Comment:
Any reason this wasn't done in the first pass?

Reply:
probably that's just an oversight